lib/uwb/UwbSession.cxx

```diff
@@ -70,3 +70,10 @@ UwbSession::StopRanging()
         return;
     }
 }
+
+void
+UwbSession::SetSessionStatus(const uwb::protocol::fira::UwbSessionStatus& status)
+{
+    PLOG_VERBOSE << "session changed state:" << m_sessionStatus.ToString() << " --> " << status.ToString();
```

Comment:
nit: there's no space between the `:` and the session value.
```suggestion
    PLOG_VERBOSE << "session changed state: " << m_sessionStatus.ToString() << " --> " << status.ToString();
```

Reply:
The session id is already being logged by the UwbSessionStatus::ToString() function so I don't think we need to do it again.

Reply:
👍